misc/dist/macos_template.app/Contents/Info.plist

```diff
@@ -24,6 +24,22 @@
 	<string>$signature</string>
 	<key>CFBundleVersion</key>
 	<string>$version</string>
+	<key>DTCompiler</key>
+	<string>com.apple.compilers.llvm.clang.1_0</string>
+	<key>DTPlatformBuild</key>
+	<string>$xcodebuild</string>
+	<key>DTPlatformName</key>
+	<string>macosx</string>
+	<key>DTPlatformVersion</key>
+	<string>$sdkver</string>
+	<key>DTSDKBuild</key>
+	<string>$sdkbuild</string>
+	<key>DTSDKName</key>
+	<string>macosx$sdkver</string>
```

Comment:
This could probably be just $sdkver as we don't know if they'll be called macosx* in the future and the DTPlatformVersion above where you've reused it doesn't seem to be needed and can be removed. If it's kept I think they should be separated as DTPlatformVersion refers to the OS version and DTSDKName to the SDK version used.

Reply:
Updated to use separate options for Platform and SDKName.